samples/python/visualization_openGL.py

```diff
@@ -15,7 +15,7 @@ def increaseTemp():
         global temperature
         temperature += 0.1
         system.thermostat.set_langevin(kT=temperature, gamma=1.0)
-        print system.temperature 
+        print temperature 
```

Comment:
`system.temperature` is depricated, 
seems like the way of choice now is 
`system.thermostat.get_state()[0]['kT']`